RefApp/ViewControllers/AssetListTableViewController.swift

```diff
@@ -28,8 +28,6 @@ class AssetListTableViewController: UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        self.navigationItem.title = NSLocalizedString("Assets", comment: "")
```

Comment:
Removed the title. But I can check the query for the loading the assets & change the title according to the asset type, if the title is needed.